src/test/java/com/rocketden/main/service/ProblemServiceTests.java

```diff
@@ -332,8 +330,7 @@ public void getRandomProblemZeroNumProblems() {
         Problem problem1 = new Problem();
         problem1.setDifficulty(ProblemDifficulty.MEDIUM);
         List<Problem> problems = Collections.singletonList(problem1);
-
-        Mockito.doReturn(problems).when(repository).findAll();
+        //Mockito.doReturn(problems).when(repository).findAllByApproval(true);
```

Comment:
I'm not sure why you removed this, so maybe re-examine that - either way, either delete the line or un-comment this out in the code.

Reply:
I wasn't sure myself, but I reran this test and got some "unncessary stubbing" error at those two lines with the `Mockito.doReturn(...)` function; I vaguely recall working through this with Alan and he said the lines weren't necessary but I'm not 100% certain about that. Either way the test passes without this line so I'll delete it for now
<img width="990" alt="Screen Shot 2021-04-05 at 10 45 24 PM" src="https://user-images.githubusercontent.com/69496997/113664453-9e317400-9660-11eb-98a5-5ea004bfbe0a.png">

Reply:
I think I still see the commented-out Mockito lines - can you remove those?

Reply:
> I think I still see the commented-out Mockito lines - can you remove those @sean-xcamp? (There are two lines I see 👍 )

Yep, thanks for pointing that out; not sure how I missed it! I included it in the next commit